src/components/table/table.tsx

```diff
@@ -75,8 +147,20 @@ export default function Table(props: TableData) {
             <TableHead>
               <TableRow>
                 {headers.map((header, index) => (
-                  <TableCell key={`header-${index}`}>
-                    {header.label || capitalizeFirstLetter(header.dataKey)}
+                  <TableCell
+                    key={`header-${index}`}
+                    //for adding aria label
+                    sortDirection={sortingCriteria[header.dataKey] || 'unset'}
+                    onClick={() => handleSortRequest(header.dataKey)}
+                  >
+                    <Tooltip title={getTooltipTitle(header.dataKey)}>
+                      <Button
```

Comment:
I think the header should not be a button, using just the TableSortLabel should be sufficient. It is already a button and has the desired styling